### PR TITLE
de.STAIR training announcement in Jena

### PR DESCRIPTION
**Pull request by bagnacan**

### CODE REVIEW

File: _events/2018-06-27-de.STAIR-primer-rna-seq.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+site: Jena
+tags:
+title: A Primer for RNA-Sequencing Processing, Interpreting and Visualization 27.-29.06.2018 in Jena
+starts: 2018-06-27
+ends: 2018-06-29
+organiser:
+  name: de.STAIR
+  email: destair@leibniz-fli.de
```
Comment (bgruening) [inline _events/2018-06-27-de.STAIR-primer-rna-seq.md]: isn't this destairdenbi@gmail.com ?

**Comment by bgruening** [comment]
Thanks @bagnacan!

@bebatut what happened to tags that are not used? Is it save to leave them in?

**Comment by bebatut** [comment]
Maybe it would be good to add the training one there: `[training]`
What do you think @bagnacan ?

File: _events/2018-06-27-de.STAIR-primer-rna-seq.md (hunk 1 above)
Comment (bagnacan) [reply _events/2018-06-27-de.STAIR-primer-rna-seq.md]: @bgruening: correct, but we're using @leibniz-fli.de for organising events and replying to participants' questions after the event is over

**Comment by bagnacan** [comment]
Ah, thanks @bebatut! I overlooked that field. I'll edit it and PR again

File: _events/2018-06-27-de.STAIR-primer-rna-seq.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+site: Jena
```
Comment (bebatut) [inline _events/2018-06-27-de.STAIR-primer-rna-seq.md]: @bagnacan You need to change it to `main`. The site here means on which sub-website it appears: `main` for https://usegalaxy-eu.github.io/ or `freiburg` for https://usegalaxy-eu.github.io/freiburg/

**Comment by bagnacan** [comment]
Thank you @bebatut!

**Comment by erasche** [comment]
Sorry for extraneous comment from jenkins bot. Looks good to me, we can merge this whenever y'all're happy.

If we want to do this future dating thing, I can set up builds on a cron job for the website so we'll know that it'll be visible on the site after that date.

**Comment by bgruening** [comment]
Thanks all.

**Comment by erasche** [comment]
Added daily build+deploy of website.

**Pull request merged**